VPC clarification for FDW (#3821)
timescale/docs: use-timescale/schema-management/foreign-data-wrappers.md

# Foreign data wrappers

You use foreign data wrappers (FDWs) to query external data sources from a $SERVICE_LONG. These external data sources can be one of the following:

- $SERVICE_LONGs
- PostgreSQL databases outside of $CLOUD_LONG

> If you are using [$VPC peering][vpc-peering], you can configure FDWs in your Customer VPC to query a $SERVICE_SHORT in Timescale $VPC or the same project. However, you can't create FDWs in Timescale $VPC to query a data source in your Customer VPC.

Reviewer: I would add something along the lines of:
This is because Timescale VPC peering uses AWS PrivateLink for increased security. Read the [VPC peering](https://docs.timescale.com/use-timescale/latest/security/vpc/) documentation for additional details.

> If you are using [$VPC peering][vpc-peering], you can configure FDWs in your Customer VPC to query a $SERVICE_SHORT in Timescale $VPC or the same project. However, you can't create FDWs in Timescale $VPC to query a data source in your Customer VPC.

Reviewer: ```suggestion
If you are using [$VPC peering][vpc-peering], you can create FDWs in your Customer VPC to query a $SERVICE_SHORT in your Timescale project. However, you can't create FDWs in your Timescale services to query a data source in your Customer VPC.
```

FDWs are particularly useful if you manage multiple $SERVICE_LONGs with different capabilities, and need to seamlessly access and merge regular and time-series data.

> ## Query another data source 

Reviewer 2: nit: trailing whitespace
